src/docs/development/platform-integration/c-interop.md

```diff
@@ -22,9 +22,10 @@ In this walkthrough, you'll create a C function
 that implements 32-bit addition and then
 exposes it through a Dart plugin named "native_add".
 
-{{ site.alert.note }}
-  The dart:ffi library is [in beta][ffi issue],
-  and breaking API changes might still happen.
+{{ site.alert.version-note }}
```

Comment:
I just checked, and flutter.dev doesn't define version-note. @sfshaza2 would you like me to add a version note like on dart.dev, or would you prefer another solution?

Reply:
We talked out-of-band and agreed that I'll add the version-note to _config.yml. (I'll do it in a separate PR.)

Reply:
PR: #5216

Reply:
Let me know when you are ready for me to review.

Reply:
Ready now. Here's a screenshot:

![image](https://user-images.githubusercontent.com/2164483/108779341-91771780-751b-11eb-9d87-43b6c803c0f2.png)